templates/app_inactive_403.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+
+{% load static %}
+<html lang="en">
+<head>
+    <title>403 - Forbidden</title>
+    <style type="text/css">
+strong {
+	color: #39F;
+}
+.Normal404Text {
+	font-family: "Times New Roman", Times, serif;
+	font-size: 30px;
+	font-style: normal;
+	font-weight: normal;
+	text-transform: none;
+	color: #39F;
+}
+    </style>
+</head>
+<body>
+    <p class="Normal404Text">403 - Forbidden</p>
+    <p class="Normal404Text">{{ detail }}</p>
+</body>
+</html>
```

Comment:
@johnfrenchxyz Can you give this a quick look over?

Reply:
@njdister I think I should clean this up in another ticket/PR. Updating the error pages has been in the back of my brain for a while.